prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -252,14 +266,21 @@ def get_storage_controller_data(self) -> Tuple[Dict[str, int], Dict[str, List]]:
         for system_id in system_ids:
             storage_controller_count[system_id] = 0
 
+            # finding storage name used in URI by querying Systems dictionary
+            # eg: if storage root URI is /redfish/v1/Systems/S1/Storages
+            # then storage_name is "Storages"
+            system_dict = self.redfish_obj.get(self.systems_root_uri + system_id).dict
```

Comment:
system_dict is a confused variable name. Maybe should relate to the essence of the return object's meaning.

Reply:
The return object is essentially a dictionary of the resources returned by querying the systems endpoint (eg: `/redfish/v1/Systems/S1`). 

Maybe a better name could be `system_resources_dict`?

Reply:
`system_resources` is better.